torch/nn/modules/module.py

```diff
@@ -232,7 +232,10 @@ def __call__(self, *input, **kwargs):
         if len(self._backward_hooks) > 0:
             var = result
             while not isinstance(var, Variable):
-                var = var[0]
+                if isinstance(var, dict):
+                    var = list(var.values())[0]
```

Comment:
Yeah that looks better. BTW You can just use `next(...)` instead of `list(...)[0]`.